widgets/nuxeo-file.html

```diff
@@ -67,19 +91,37 @@
       </dom-if>
     </div>
 
-    <dom-repeat items="[[files]]" as="file">
+    <label class="error" hidden$="[[!invalid]]">[[errorMessage]]</label>
+
+    <dom-if if="[[!multiple]]">
+      <template>
+        <dom-if if="[[value]]">
+          <template>
+            <div class="file">
+              <div class="layout horizontal">
+                <a href$="[[value.data]]">[[_fileName(value)]]</a>
```

Comment:
Use the `download` attribute to make sure the file is downloaded and no navigation occurs. This is crucial for downloading from urls generated from our code (see details below). Optionally, we can pass the file name to the download attribute to make sure it always gets a name and no weird ids are used:
```suggestion
                <a href$="[[value.data]]" download="[[_fileName(value)]]">[[_fileName(value)]]</a>
```